unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/InputController_Legacy.cs

```diff
@@ -50,6 +50,26 @@ private InputController_Legacy()
+            
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.ForwardButtonKeyCode, buttonId = WebInterface.ACTION_BUTTON.FORWARD, useRaycast = true });
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.ForwardButtonKeyCodeAlt, buttonId = WebInterface.ACTION_BUTTON.FORWARD, useRaycast = true });
+            
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.BackwardButtonKeyCode, buttonId = WebInterface.ACTION_BUTTON.BACKWARD, useRaycast = true });
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.BackwardButtonKeyCodeAlt, buttonId = WebInterface.ACTION_BUTTON.BACKWARD, useRaycast = true });
+            
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.RightButtonKeyCode, buttonId = WebInterface.ACTION_BUTTON.RIGHT, useRaycast = true });
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.RightButtonKeyCodeAlt, buttonId = WebInterface.ACTION_BUTTON.RIGHT, useRaycast = true });
+            
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.LeftButtonKeyCode, buttonId = WebInterface.ACTION_BUTTON.LEFT, useRaycast = true });
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.LeftButtonKeyCodeAlt, buttonId = WebInterface.ACTION_BUTTON.LEFT, useRaycast = true });
+            
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.WalkButtonKeyCode, buttonId = WebInterface.ACTION_BUTTON.WALK, useRaycast = true });
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.JumpButtonKeyCode, buttonId = WebInterface.ACTION_BUTTON.JUMP, useRaycast = true });
+            
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.OneButtonKeyCode, buttonId = WebInterface.ACTION_BUTTON.ACTION_1, useRaycast = true });
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.TwoButtonKeyCode, buttonId = WebInterface.ACTION_BUTTON.ACTION_2, useRaycast = true });
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.ThreeButtonKeyCode, buttonId = WebInterface.ACTION_BUTTON.ACTION_3, useRaycast = true });
+            buttonsMap.Add(new BUTTON_MAP() { type = BUTTON_TYPE.KEYBOARD, buttonNum = (int) InputSettings.FourButtonKeyCode, buttonId = WebInterface.ACTION_BUTTON.ACTION_4, useRaycast = true });
```

Comment:
what are the `useRaycast` used for?

Reply:
`useRaycast` determines if the event should be sent with the raycast information like: hit point, hit normal, hit distance and hit entityId

Reply:
Is that needed? I'm thinking about how much it will cost every frame the key is pressed